data/publiccloud/terraform/ec2.tf

```diff
@@ -17,7 +17,7 @@ variable "image_id" {
 }
 
 resource "random_id" "service" {
-    count = "${var.count}"
+    count = "${var.instance_count}"
     keepers {
```

Comment:
Why we do not need to add a `=` here?

Reply:
I added it after Terraform 0.12 complained that it needed a '='. 

```
# terraform plan -no-color -var 'image_id=sles12sp5gcex8664010byosbuild118' -var 'instance_count=1' -var 'type=n1-standard-2' -var 'region=europe-west1-b' -var 'name=cfconrad-vm' -out myplan ; echo V74GK-$?-

Error: Unsupported block type

  on plan.tf line 41, in resource "random_id" "service":
  41:     keepers {

Blocks of type "keepers" are not expected here. Did you mean to define
argument "keepers"? If so, use the equals sign to assign it a value.
```

Reply:
I don't get it . So we do need a `=` sign for gce, but not with ec2 and azure?
And all within the same resource `resource "random_id" "service"`, that sounds inconsistent!

Reply:
because I didn't test ec2 and azure when the first patch was uploaded. See now the 3 .tf's

Reply:
makes sense now